server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/issues/IssuesPage.java

```diff
@@ -57,30 +53,30 @@ public IssuesPage issuesCount(Integer count) {
   }
 
   public Issue getFirstIssue() {
-    getIssuesElements().shouldHave(sizeGreaterThan(0));
+    getIssuesElements().shouldHave(CollectionCondition.sizeGreaterThan(0));
     return new Issue(getIssuesElements().first());
   }
 
   public IssuesPage componentsShouldContain(String path) {
-    this.getIssuesPathComponents().forEach(element -> element.shouldHave(text(path)));
+    this.getIssuesPathComponents().forEach(element -> element.shouldHave(Condition.text(path)));
     return this;
   }
 
   public IssuesPage componentsShouldNotContain(String path) {
-    this.getIssuesPathComponents().forEach(element -> element.shouldNotHave(text(path)));
+    this.getIssuesPathComponents().forEach(element -> element.shouldNotHave(Condition.text(path)));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)